src/core/sys/posix/aio.d

```diff
@@ -0,0 +1,66 @@
+/**
+ * D header file to interface with the
+ * $(HTTP pubs.opengroup.org/onlinepubs/9699919799/basedefs/aio.h.html, Linux aio API).
+ * Available since Linux 2.6
+ *
+ * Copyright: Copyright D Language Foundation 2018.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   $(HTTPS github.com/darredevil, Alexandru Razvan Caciulescu)
+ */
+module core.sys.posix.aio;
+
+private import core.sys.posix.signal;
+
+version (CRuntime_Glibc):
+version (X86_64):
```

Comment:
This should not be here, rather it should be moved to where x86_64-specific definitions are.